platform/contracts/admin/src/endpoints.rs

```diff
@@ -209,7 +212,7 @@ pub fn query(deps: Deps<'_>, env: Env, msg: QueryMsg) -> ContractResult<Binary>
 
             let addr = deps.api.addr_humanize(&canonical_addr)?;
 
-            sdk::cosmwasm_std::to_json_binary(&addr).map_err(From::from)
+            to_json_binary(&addr).map_err(From::from)
```

Comment:
```suggestion
            cosmwasm_std::to_json_binary(&addr).map_err(From::from)
```

Reply:
following the practice of referring functions by their package

Reply:
I was it was imported and didn't give it a second thought. Corrected now.